oresat_dxwifi/services/camera.py

```diff
@@ -0,0 +1,95 @@
+"""Oresat Live Camera Service"""
+
+from olaf import Service, logger
+from enum import IntEnum
+from ..camera.camera import CameraInterface
+
+
+class State(IntEnum):
+    OFF = 0
+    BOOT = 1
+    STANDBY = 2
+    FILMING = 3
+    ERROR = 0xFF
+
+
+STATE_TRANSISTIONS = {
+    State.OFF: [State.BOOT],
+    State.BOOT: [State.STANDBY],
```

Comment:
Does `State.BOOT` really do anything for us? I see that `CameraService.on_start()` sets state to `State.STANDBY`. Wondering if both `OFF` and `BOOT` are necessary, or if we can use just one.

Reply:
I was following how star-tracker implemented theirs. I think it is fine to leave for now because it leaves a distinction between the states in case we need to add more to what they do in the future.